workers.rb

```diff
@@ -482,7 +482,7 @@ def perform(nwo, issue_id, config, dry_run)
     if dry_run == true
       pr_url = create_deposit_pr(issue_id, config.papers_repo, config.journal_alias, dry_run)
 
-      pr_response = "Check final proof :point_right: #{pr_url}\n\nIf the paper PDF and Crossref deposit XML look good in #{pr_url}, then you can now move forward with accepting the submission by compiling again with the flag `deposit=true` e.g.\n ```\n@whedon accept deposit=true\n```"
+      pr_response = ":wave: @#{config.eic_team_name}, this paper is ready to be accepted and published\n\n. Check final proof :point_right: #{pr_url}\n\nIf the paper PDF and Crossref deposit XML look good in #{pr_url}, then you can now move forward with accepting the submission by compiling again with the flag `deposit=true` e.g.\n ```\n@whedon accept deposit=true\n```"
```

Comment:
When the editor runs `@whedon accept`, Whedon mentions the EiC team to bring the paper to their attention.